source/connect/connections.txt

```diff
@@ -0,0 +1,81 @@
+.. _compass-sidebar:
+
+===============
+Compass Sidebar
```

Comment:
Thoughts on calling this "Connections Sidebar?" We're really only discussing a portion of the Compass sidebar, and you use "Connections sidebar" in other parts of the docs.

Reply:
Done!